tests/test_sample_simple_fps.py

```diff
@@ -31,9 +31,9 @@ def test_initialize(self):
         and throws an error otherwise
         """
 
-        for initialize in [self.idx[0], "random"]:
+        for initialize in [self.idx[0], "random", self.idx[:4]]:
```

Comment:
we should also have a test that restarting from a list gives back the indexes as the first selected values

Reply:
I pulled the test for the initialize list out of this loop to simplify things (remove the need for `len(idx) - 1`) and add the additional check that the lists are the same.